authorizer-app-backend/src/main/java/org/radarbase/authorizer/service/OAuth1RestSourceAuthorizationService.kt

```diff
@@ -96,17 +103,14 @@ abstract class OAuth1RestSourceAuthorizationService(
         return Url(authConfig.authorizationEndpoint, params).getUrl()
     }
 
-    fun requestToken(tokenEndpoint: String?, tokens: RestOauth1AccessToken, sourceType: String): RestOauth1AccessToken? {
+    fun requestToken(tokenEndpoint: String?, tokens: RestOauth1AccessToken, sourceType: String): RestOauth1AccessToken {
         val req = createRequest("POST", tokenEndpoint.orEmpty(), tokens, sourceType)
 
         return httpClient.newCall(req).execute().use { response ->
             when (response.code) {
                 200 -> response.body?.string()
                         ?.let { tokenReader.readValue<RestOauth1AccessToken>(parseParams(it)) }
                         ?: throw HttpBadGatewayException("Service did not provide a result")
-                400, 401, 403 ->{
-                    logger.info(response.body?.string())
-                    null }
```

Comment:
It might be good to keep the return null here: it ensures that a user is marked as unauthorised once this state occurs so someone needs to intervene to get new credentials for the user.